messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQServerService.java

```diff
@@ -76,7 +76,7 @@
  * @author scott.stark@jboss.org
  * @author Emanuel Muckenhuber
  */
-class ActiveMQServerService implements Service<ActiveMQServer> {
+class ActiveMQServerService implements Service<ActiveMQServer>, Supplier<ActiveMQServer> {
```

Comment:
Why implement Supplier? I don't see any use.

Reply:
Also, change this to org.jboss.msc.Service and drop the getValue() method.

Reply:
Never mind this last part; I forgot about the ActiveMQActivationService.isActiveMQServerActive thing.